AsyncDisplayKit/ASCellNode.h

```diff
@@ -122,6 +124,12 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
 
 @end
 
+@interface ASCellNode (Deprecated)
```

Comment:
Could you add a comment above this saying:

// Previous versions of ASDK did not include "is" in the name of the getter for these properties.
// These older accessor methods don't match UIKit naming, and will be removed in a future version.